apps/shipments/views/shipment_overview.py

```diff
@@ -87,16 +88,20 @@ class ShipmentOverviewListView(jsapi_views.PreloadIncludesMixin,
     filter_backends = (InBBoxFilter, filters.SearchFilter, filters.OrderingFilter, DjangoFilterBackend,)
     bbox_filter_field = 'point'
     search_fields = tuple([f'shipment__{field}' for field in SHIPMENT_SEARCH_FIELDS])
-    ordering_fields = ('shipment__created_at', )
+    ordering_fields = tuple([f'shipment__{field}' for field in SHIPMENT_ORDERING_FIELDS])
     filterset_class = ShipmentOverviewFilter
 
     def dispatch(self, request, *args, **kwargs):
         # pylint:disable=protected-access
         request.GET._mutable = True  # Make query_params mutable
         for param in dict(request.GET):
             # Prepend non-trackingdata search/filter/order query params with 'shipment__' to properly query relation
-            if param not in ('search', 'in_bbox',) and request.GET.getlist(param):
+            if param not in ('search', 'in_bbox', 'ordering') and request.GET.getlist(param):
                 request.GET.setlist(f'shipment__{param}', request.GET.pop(param))
+            elif param in ('ordering',) and request.GET.getlist(param):
+                request.GET.setlist(param,
+                                    [f'-shipment__{ship_ordering.replace("-", "")}' if '-' in ship_ordering else
```

Comment:
Prefer ship_ordering.startswith('-'), it should be faster and is what you are actually looking for.

Could also condense somewhat:
`f'{"-" if ship_ordering.startswith("-") else ""}shipment__{ship_ordering}'`

Reply:
we can use the startswith but I'm unsure of the condensing as we need to also ensure that the '-' is removed from the ship_ordering